README.md

```diff
@@ -60,7 +60,20 @@ dism { 'Internet-Explorer-Optional-amd64':
 ~~~
 
 ##Reference
-
+###Deployment on Azure Windows Server 2012 R2 Datacentre (Image from Azure Store)
+####Windows Server 2012 R2 Datacentre does not have dism installed under c:\windows\sysnative 
+####it is under c:\windows\system32
```

Comment:
Hi
Thanks Rob, when you asked me what version of DISM i used then i realized
of my mistake, the version i had in that env was too old, 0.2.0 and when
updated, all worked fine.
The ruby reference for the alternate path works.
I have closed the merge request.
so sorry for this
BR
Daniel.

On Thu, Oct 22, 2015 at 8:04 PM, Rob Reynolds <notifications@github.com>
wrote:

> In README.md
> <https://github.com/puppetlabs/puppetlabs-dism/pull/37#discussion_r42783274>
> :
>
> > @@ -60,7 +60,20 @@ dism { 'Internet-Explorer-Optional-amd64':
> >  ~~~
> >
> >  ##Reference
> > -
> > +###Deployment on Azure Windows Server 2012 R2 Datacentre (Image from Azure Store)
> > +####Windows Server 2012 R2 Datacentre does not have dism installed under c:\windows\sysnative
> > +####it is under c:\windows\system32
>
> Also, if you take a look here at
> https://github.com/puppetlabs/puppetlabs-dism/blob/master/lib/puppet/provider/dism/dism.rb#L7-L14,
> you will notice we search for the command in several places.
>
> What version of the DISM module do you have?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/puppetlabs/puppetlabs-dism/pull/37/files#r42783274>.
>